libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -76,6 +76,36 @@
     "days": "{{ days }}d {{ duration.hours }}"
   },
   "shared": {
+    "executionTypes": {
+      "EXTERNAL": "Ekstern",
+      "INTERNAL": "Intern"
+    },
+    "steps": {
+      "0": {
+        "pending": "Afventer planlægning",
+        "running": "Planlægger",
+        "failed": "Planlægning fejlet",
+        "skipped": "Planlægning sprunget over",
+        "canceled": "Planlægning annulleret",
+        "succeeded": "Planlagt"
+      },
+      "1": {
+        "pending": "Afventer beregning",
+        "running": "Beregner",
+        "failed": "Beregning fejlet",
+        "skipped": "Beregning sprunget over",
+        "canceled": "Beregning annulleret",
+        "succeeded": "Beregnet"
+      },
+      "2": {
+        "pending": "Afventer beskeddannelse",
+        "running": "Danner beskeder",
+        "failed": "Beskeddannelse fejlet",
+        "skipped": "Beskeddannelse sprunget over",
+        "canceled": "Beskeddannelse annulleret",
+        "succeeded": "Beskeder dannet"
+      }
+    },
```

Comment:
suggestion: these are specific to calculations and should not be considered shared

Reply:
Oooh, do we have some more generic naming for a process then?

Reply:
No, steps are always specific to individual "processes", meaning the number of steps can be different between processes and fx. step 1 is not always a "calculation" step, but could be anything. So this translation only applies to calculation processes and should not be used for requests. I don't actually know what the steps for a request are, so maybe we should just only show steps for calculations (for now)? And then move these translations back into the "wholesale.calculations" area.

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3958/commits/6611bd5f0e0156e5eafdd2c035af715ce39b8087